src/core/core.scale.js

```diff
@@ -1561,6 +1561,9 @@ export default class Scale extends Element {
     if (area) {
       unclipArea(ctx);
     }
+
+    return items;
```

Comment:
This feels kind of weird, function name is `drawLabels` so I would expect it to draw labels not give them to me

Reply:
I agree that returning from the draw* function was weird - rather than implement a new top-level API, I implemented a new public Scale function: getLabelItems(chartArea), which I think suits the needs pretty well. I'll provide a code-pen asap that demos the new code.